howto/allowlists-serviceinvocation/README.md

```diff
@@ -0,0 +1,117 @@
+# Access Control List for service invocation
+
+Access control enables the configuration of policies that restrict what operations *calling* applications can perform, via service invocation, on the *called* application. To limit access to a called applications from specific operations and HTTP verbs from the calling applications, you can define an access control policy specification in configuration.
+
+Below are some example scenarios for using access control list for service invocation. See [configuration guidance](../../concepts/configuration/README.md) to understand the available configuration settings for an application sidecar.
+
+## Scenario 1 : Deny access to all apps except where trustDomain = public, namespace = default, appId = app1
+With this configuration, all calling methods with appId = app1 are allowed and all other invocation requests from other applications are denied
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+spec:
+  accessControl:
+    defaultAction: deny
+    trustDomain: "public"
+    policies:
+    - app: app1
+      defaultAction: allow
+      trustDomain: 'public'
+      namespace: "default"
+```
+
+## Scenario 2 : Deny access to all apps except trustDomain = public, namespace = default, appId = app1, operation = op1
+With this configuration, only method op1 from appId = app1 is allowed and all other method requests from all other apps, including other methods on app1, are denied
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+spec:
+  accessControl:
+    defaultAction: deny
+    trustDomain: "public"
+    policies:
+    - app: app1
+      defaultAction: deny
+      trustDomain: 'public'
+      namespace: "default"
+      operations:
+      - name: /op1
+        httpVerb: ['*']
+        action: allow
+```
+
+## Scenario 3 : Deny access to all apps except trustDomain = public, namespace = default, appID = app1, operation = op1, http verb = POST/PUT
+With this configuration, only HTTP verbs POST/PUT on method op1 from appId = app1 are allowed and all other method requests from all other apps, including other methods on app1, are denied
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+spec:
+  accessControl:
+    defaultAction: deny
+    trustDomain: "public"
+    policies:
+    - app: app1
+      defaultAction: deny
+      trustDomain: 'public'
+      namespace: "default"
+      operations:
+      - name: /op1
+        httpVerb: ['POST', 'PUT']
+        action: allow
+```
+
+## Scenario 4 :
```

Comment:
What is this scenario? Please do not delete this, since this looked to actually be a useful example that can be applied to the hello world app and I would really like to keep this. However did you mean to put two apps into the same configuration file?

Reply:
This was actually a cut-paste error. I think what we need for hello-world app is covered by Scenario 2. We just need to change the app and method names